.travis.yml

```diff
@@ -45,11 +45,13 @@ script:
       cd ..
       git clone https://github.com/QCoDeS/qcodes_generate_test_db.git
       cd qcodes_generate_test_db
+      git checkout bumping-to-version-5
```

Comment:
I assume that this should go and the branch merged before this goes in?

Reply:
absolutely (see todo above)